# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: ddm 0.7.1\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2014-12-03 16:41+0100\n"
"PO-Revision-Date: 2013-12-26 18:07+0200\n"
"Last-Translator: Taylan Tatlı <taylantatli90@gmail.com>\n"
"Language-Team: SolydXK <admin@solydxk.com>\n"
"Language: tr\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Generator: Poedit 1.5.4\n"
"X-Poedit-SourceCharset: UTF-8\n"

#: ../usr/lib/ddm/ati.py:73
#, python-format
msgid "ATI card found: %(card)s"
msgstr "ATI kart bulundu: %(card)s"

#: ../usr/lib/ddm/ati.py:78
#, python-format
msgid "ATI chip serie found: %(serie)s"
msgstr "ATI çip serisi bulundu: %(serie)s"

#: ../usr/lib/ddm/ati.py:87
#, python-format
msgid "ATI driver found: %(drv)s (%(status)s)"
msgstr "ATI sürücüsü bulundu: %(drv)s (%(status)s)"

#: ../usr/lib/ddm/ati.py:90 ../usr/lib/ddm/via.py:56
#: ../usr/lib/ddm/intel.py:57 ../usr/lib/ddm/nvidia.py:104
#, python-format
msgid "Driver not installable: %(drv)s"
msgstr "Sürücü kurulabilir değil: %(drv)s"

#: ../usr/lib/ddm/ati.py:92
#, python-format
msgid "ATI chip serie not supported: %(serie)d"
msgstr "ATI çip serisi desteklenmiyor: %(serie)d"

#: ../usr/lib/ddm/ati.py:94
#, python-format
msgid "No driver found for: %(card)s (%(man)s:%(serie)s)"
msgstr "Bunun için sürücü bulunamadı: %(card)s (%(man)s:%(serie)s)"

#: ../usr/lib/ddm/ati.py:107 ../usr/lib/ddm/via.py:68
#: ../usr/lib/ddm/intel.py:69 ../usr/lib/ddm/nvidia.py:118
#, python-format
msgid "Install driver: %(drv)s"
msgstr "Sürücüyü yükle: %(drv)s"

#: ../usr/lib/ddm/ati.py:121 ../usr/lib/ddm/nvidia.py:144
#, python-format
msgid "Switch to module: %(module)s"
msgstr "Modüle geçiş yap: %(module)s"

#: ../usr/lib/ddm/ati.py:127 ../usr/lib/ddm/via.py:77
#: ../usr/lib/ddm/intel.py:78 ../usr/lib/ddm/nvidia.py:154
#, python-format
msgid "Driver installed: %(drv)s"
msgstr "Sürücü yüklendi: %(drv)s"

#: ../usr/lib/ddm/ati.py:138
msgid "Removing ATI drivers"
msgstr "ATI sürücüleri kaldırılıyor"

#: ../usr/lib/ddm/ati.py:152 ../usr/lib/ddm/via.py:96
#: ../usr/lib/ddm/intel.py:97 ../usr/lib/ddm/nvidia.py:183
#, python-format
msgid "Driver removed: %(drv)s"
msgstr "Sürücü kaldırıldı: %(drv)s"

#: ../usr/lib/ddm/ati.py:160
msgid "ATI display driver"
msgstr "ATI görüntü sürücüsü"

#: ../usr/lib/ddm/ati.py:162 ../usr/lib/ddm/intel.py:105
#: ../usr/lib/ddm/nvidia.py:193
msgid "Intel display driver"
msgstr "Intel görüntü sürücüsü"

#: ../usr/lib/ddm/ati.py:164
msgid "Radeon display driver"
msgstr "Radeon görüntü sürücüsü"

#: ../usr/lib/ddm/ati.py:166 ../usr/lib/ddm/via.py:106
#: ../usr/lib/ddm/intel.py:107 ../usr/lib/ddm/nvidia.py:197
msgid "Framebuffer display driver"
msgstr "Framebuffer görüntü sürücüsü"

#: ../usr/lib/ddm/ati.py:168 ../usr/lib/ddm/via.py:108
#: ../usr/lib/ddm/intel.py:109 ../usr/lib/ddm/nvidia.py:199
msgid "Vesa display driver"
msgstr "Vesa görüntü sürücüsü"

#: ../usr/lib/ddm/broadcom.py:89
#, python-format
msgid "Broadcom chip serie found: %(chip)s"
msgstr "Broadcom çip serisi bulundu: %(chip)s"

#: ../usr/lib/ddm/broadcom.py:158
#, python-format
msgid "Used wireless driver: %(drv)s"
msgstr "Kullanılan kablosuz ağ sürücüsü: %(drv)s"

#: ../usr/lib/ddm/broadcom.py:179
#, python-format
msgid "Broadcom found: %(broadcom)s"
msgstr "Broadcom bulundu: %(broadcom)s"

#: ../usr/lib/ddm/broadcom.py:202
#, python-format
msgid "Broadcom chipset not supported or ethernet controller: %(broadcom)s"
msgstr ""
"Broadcom çipseti veya ethernet kontrolcüsü desteklenmiyor: %(broadcom)s"

#: ../usr/lib/ddm/broadcom.py:204
#, python-format
msgid "Broadcom chipset not found: %(chip)s"
msgstr "Broadcom çipseti bulunamadı: %(chip)s"

#: ../usr/lib/ddm/broadcom.py:207
#, python-format
msgid "Broadcom pci ID not found: %(id)s"
msgstr "Broadcom pci ID bulunamadı: %(id)s"

#: ../usr/lib/ddm/broadcom.py:216
#, python-format
msgid "Linux header name to install: %(header)s"
msgstr "Kurulucak Linux header adı: %(header)s"

#: ../usr/lib/ddm/broadcom.py:220
#, python-format
msgid "Download package: %(package)s"
msgstr "Paketi indir: %(package)s"

#: ../usr/lib/ddm/broadcom.py:228
#, python-format
msgid "Download package: %(drv)s"
msgstr "Paketi indir: %(drv)s"

#: ../usr/lib/ddm/broadcom.py:265
msgid "Install downloaded packages"
msgstr "İndirilmiş paketleri kur"

#: ../usr/lib/ddm/broadcom.py:273
#, python-format
msgid "Reconfigure package: %(package)s"
msgstr "Paketi yeniden yapılandır:  %(package)s"

#: ../usr/lib/ddm/broadcom.py:316
#, python-format
msgid "Purge driver: %(drv)s"
msgstr "Sürücüyü kaldır: %(drv)s"

#: ../usr/lib/ddm/broadcom.py:326
msgid "Done removing Broadcom drivers"
msgstr "Broadcom sürücüsünü kaldırma işlemi tamamlandı"

#: ../usr/lib/ddm/xorg.py:81
#, python-format
msgid "Used graphics driver: %(module)s"
msgstr "Kullanılan grafik sürücüsü: %(module)s"

#: ../usr/lib/ddm/xorg.py:103
msgid "No xorg.conf - create minimal xorg.conf"
msgstr "xorg.conf dosyası yok - minimal bir xorg.conf dosyası yarat"

#: ../usr/lib/ddm/xorg.py:123
msgid "New xorg.conf written"
msgstr "Yeni xorg.conf dosyası yazıldı"

#: ../usr/lib/ddm/xorg.py:129
msgid "New section appended to xorg.conf"
msgstr "Yeni bölüm xorg.conf dosyasına eklendi"

#: ../usr/lib/ddm/via.py:47
#, python-format
msgid "Via card found: %(card)s"
msgstr "Via kartı bulundu: %(card)s"

#: ../usr/lib/ddm/via.py:53
#, python-format
msgid "Via driver found: %(drv)s (%(status)s)"
msgstr "Via sürücüsü bulundu: %(drv)s (%(status)s)"

#: ../usr/lib/ddm/via.py:104
msgid "Chrome display driver"
msgstr "Chrome görüntü sürücüsü"

#: ../usr/lib/ddm/intel.py:48
#, python-format
msgid "Intel card found: %(card)s"
msgstr "Intel kart bulundu: %(card)s"

#: ../usr/lib/ddm/intel.py:54
#, python-format
msgid "Intel driver found: %(drv)s (%(status)s)"
msgstr "Intel sürücüsü bulundu: %(drv)s (%(status)s)"

#: ../usr/lib/ddm/ddm.py:72
msgid "Graphics"
msgstr "Grafik"

#: ../usr/lib/ddm/ddm.py:73
msgid "Wireless"
msgstr "Kablosuz"

#: ../usr/lib/ddm/ddm.py:74
msgid "Kernel"
msgstr "Kernel"

#: ../usr/lib/ddm/ddm.py:75
msgid "Hardware"
msgstr "Donanım"

#: ../usr/lib/ddm/ddm.py:76
msgid "Activated driver"
msgstr "Aktif sürücü"

#: ../usr/lib/ddm/ddm.py:77
msgid "Alternative drivers"
msgstr "Alternatif Sürücüler"

#: ../usr/lib/ddm/ddm.py:116 ../usr/lib/ddm/ddm.py:147
msgid "Current drivers are correct"
msgstr "Mevcut sürücüler doğru"

#: ../usr/lib/ddm/ddm.py:157
msgid "PAE already installed"
msgstr "PAE zaten yüklenmiş"

#: ../usr/lib/ddm/ddm.py:159
msgid "PAE support only for 32-bit systems"
msgstr "PAE desteği sadece 32-bit sistemler için geçerlidir"

#: ../usr/lib/ddm/ddm.py:199
msgid "Activate"
msgstr "Etkinleştir"

#: ../usr/lib/ddm/ddm.py:199
msgid "Driver"
msgstr "Sürücü"

#: ../usr/lib/ddm/ddm.py:199
msgid "Version"
msgstr "Sürüm"

#: ../usr/lib/ddm/ddm.py:199
msgid "Description"
msgstr "Açıklama"

#: ../usr/lib/ddm/ddm.py:214
msgid " (recommended)"
msgstr " (önerilir)"

#: ../usr/lib/ddm/ddm.py:259
msgid "Unknown driver"
msgstr "Bilinmeyen Sürücüsü"

#: ../usr/lib/ddm/ddm.py:262
msgid ""
"Unknown driver found.\n"
"\n"
"Please remove before installing drivers with DDM."
msgstr ""
"Bilinmeyen sürücü bulundu.\n"
"\n"
"Lütfen DDM ile sürücüleri yüklemeden önce kaldırın."

#: ../usr/lib/ddm/ddm.py:307
msgid "Driver activation"
msgstr "Sürücü etkinleştirme"

#: ../usr/lib/ddm/ddm.py:307
#, python-format
msgid ""
"Are you sure you want to install the following driver:\n"
"\n"
"%(drv)s"
msgstr ""
"Bu sürücüyü yüklemek istediğinize emin misiniz:\n"
"\n"
"%(drv)s"

#: ../usr/lib/ddm/ddm.py:312 ../usr/lib/ddm/ddm.py:452
msgid "Locked"
msgstr "Kilitlendi"

#: ../usr/lib/ddm/ddm.py:312
#, python-format
msgid ""
"Could not get lock on %(file)s.\n"
"\n"
"Close any programs locking the file and try again."
msgstr ""
"%(file)s dosyasından kilit alınamadı.\n"
"\n"
"Dosyayı kilitleyen uygulamaları kapatın ve tekrar deneyin."

#: ../usr/lib/ddm/ddm.py:390
msgid "Driver installed"
msgstr "Sürücü kuruldu"

#: ../usr/lib/ddm/ddm.py:390
#, python-format
msgid ""
"Driver installed: %(drv)s\n"
"\n"
"Drivers will be used on next boot.\n"
"Do you want to reboot now?"
msgstr ""
"Sürücü kuruldu: %(drv)s\n"
"\n"
"Sürücüler bir sonraki açılışta kullanılacak.\n"
"Şimdi yeniden başlatmak ister misiniz?"

#: ../usr/lib/ddm/ddm.py:452
#, python-format
msgid ""
"Could not get lock on %(file)s.\n"
"\n"
"Close any programs locking the file and restart DDM."
msgstr ""
"%(file)s dosyasından kilit alınamadı.\n"
"\n"
"Dosyayı kilitleyen uygulamaları kapatın ve DDM'yi yeniden başlatın."

#: ../usr/lib/ddm/ddm.py:485
msgid "Checking your hardware..."
msgstr "Donanım bilgileri alınıyor..."

#: ../usr/lib/ddm/ddm.py:550
msgid "Start automatic driver install"
msgstr "Otomatik sürücü kurulumunu başlat"

#: ../usr/lib/ddm/pae.py:43
#, python-format
msgid "Installed PAE: %(pae)s"
msgstr "Kurulu PAE: %(pae)s"

#: ../usr/lib/ddm/pae.py:49
msgid "Multi-core support for 32-bit systems"
msgstr "32-bit sistemler için çoklu-çekirdek desteği"

#: ../usr/lib/ddm/pae.py:59
#, python-format
msgid "Multi-core already installed: %(package)s"
msgstr "Çoklu-çekirdek zaten yüklü: %(package)s"

#: ../usr/lib/ddm/pae.py:60 ../usr/lib/ddm/pae.py:73
msgid "PAE capable system"
msgstr "PAE destekli sistem"

#: ../usr/lib/ddm/pae.py:66
msgid "Multi-core system running single-core kernel found"
msgstr "Çok-çekirdekli sistemin tek-çekirdek kernel çalıştırdığı bulundu"

#: ../usr/lib/ddm/pae.py:70
msgid "PAE not installed"
msgstr "PAE kurulu değil"

#: ../usr/lib/ddm/pae.py:79
msgid "PAE kernel cannot be installed: single-core system"
msgstr "PAE kernel kurulamadı: tek-çekirdekli sistem"

#: ../usr/lib/ddm/pae.py:95
msgid "Done installing PAE"
msgstr "PAE kurulumu tamamlandı"

#: ../usr/lib/ddm/pae.py:109
#, python-format
msgid "PAE package to remove: %(package)s"
msgstr "Kaldırılacak PAE paketi: %(package)s"

#: ../usr/lib/ddm/pae.py:112
msgid "Done removing PAE"
msgstr "PAE kaldırma işlemi tamamlandı"

#: ../usr/lib/ddm/pae.py:114
msgid "Cannot remove PAE when running PAE"
msgstr "PAE kullanımdayken kaldırılamaz"

#: ../usr/lib/ddm/nvidia.py:57
msgid "Update apt"
msgstr "APT güncelle"

#: ../usr/lib/ddm/nvidia.py:60
msgid "Install nvidia-detect"
msgstr "nvidia-detect'i kur"

#: ../usr/lib/ddm/nvidia.py:68
msgid "Cannot install nvidia-detect: abort"
msgstr "nvidia-detect kurulamıyor: durduruldu"

#: ../usr/lib/ddm/nvidia.py:92
#, python-format
msgid "Nvidia card found: %(card)s"
msgstr "Nvidia kartı bulundu: %(card)s"

#: ../usr/lib/ddm/nvidia.py:101
#, python-format
msgid "Nvidia driver found: %(drv)s (%(status)s)"
msgstr "Nvidia sürücüsü bulundu: %(drv)s (%(status)s)"

#: ../usr/lib/ddm/nvidia.py:166
msgid "Removing Nvidia drivers"
msgstr "Nvidia sürücüleri kaldırılıyor"

#: ../usr/lib/ddm/nvidia.py:191
msgid "Nvidia display driver"
msgstr "Nvidia görüntü sürücüsü"

#: ../usr/lib/ddm/nvidia.py:195
msgid "Nouveau display driver"
msgstr "Nouveau görüntü sürücüsü"

#: ../usr/lib/ddm/main.py:67
#, python-format
msgid "Write debug information to file: %(path)s"
msgstr "Hata ayıklama bilgisini dosyaya yaz: %(path)s"

#: ../usr/lib/ddm/main.py:71
#, python-format
msgid "Machine info: %(info)s"
msgstr "Makine bilgisi: %(info)s"

#: ../usr/lib/ddm/main.py:73
#, python-format
msgid "DDM version: %(version)s"
msgstr "DDM sürümü: %(version)s"

#: ../usr/lib/ddm/main.py:78
msgid "apt-listbugs is installed and might interfere with driver installation"
msgstr "apt-listbugs kuruldu ve sürücü kurulumunu engelleyebilir"

#: ../usr/lib/ddm/main.py:108 ../usr/lib/ddm/main.py:110
msgid "Please enter your password"
msgstr "Lütfen parolanızı girin"

#: ../usr/lib/ddm/main.py:116
msgid "DDM - Kernel"
msgstr "DDM - Kernel"

#: ../usr/lib/ddm/main.py:117
msgid ""
"You do not have the latest kernel installed.\n"
"\n"
"Use the Update Manager to update your system."
msgstr ""
"Son sürüm kernel'a sahip değilsiniz.\n"
"\n"
"Sisteminizi güncellemek için Güncelleme Yöneticisini kullanın."

#: ../usr/lib/ddm/main.py:121
msgid "DDM - Live environment"
msgstr "DDM - Canlı ortam"

#: ../usr/lib/ddm/main.py:122
msgid ""
"DDM cannot run in a live environment\n"
"\n"
"To force start, use the --force argument"
msgstr ""
"DDM canlı bir ortamda çalıştırılamaz\n"
"\n"
"Zorla başlatmak için,  --force değişkenini kullanın"

#: ../usr/lib/ddm/main.py:126
msgid "DDM - Debian based"
msgstr "DDM - Debian tabanlı"

#: ../usr/lib/ddm/main.py:127
msgid ""
"Cannot determine the base distribution (debian or ubuntu)\n"
"\n"
"To force start, use the --force argument"
msgstr ""
"Dağıtım tabanı belirlenemedi (debian veya ubuntu)\n"
"\n"
"Zorla başlatmak için,  --force değişkenini kullanın"

#: ../usr/lib/ddm/main.py:131
msgid "DDM - Internet"
msgstr "DDM - İnternet"

#: ../usr/lib/ddm/main.py:132
msgid ""
"You do not seem to have an internet connection\n"
"\n"
"To force start, use the --force argument"
msgstr ""
"Bir internet bağlantısı var gibi görünmüyor\n"
"\n"
"Zorla başlatmak için,  --force değişkenini kullanın"

#: ../usr/share/applications/ddm.desktop.in.h:1
msgid "Device Driver Manager"
msgstr ""

#~ msgid "Install aborted: not in repository: %(repo)s"
#~ msgstr "Kurulum durduruldu: depoda yok: %(repo)s"
